libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnCondition.schema

```diff
@@ -19,6 +19,15 @@
             "items": {
                 "$kind": "Microsoft.IDialog"
             }
+        },
+        "priority": {
+            "type": "string",
+            "description": "Priority expression of rule with 0 being the most important",
+            "$role": "expression"
+        },
+        "runOnce": {
+            "type": "boolean",
+            "description": "True if rule should run once per unique conditions"
         }
```

Comment:
do all of the conditions include this definition?

Reply:
Yes.

---
In reply to: [360176011](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360176011) [](ancestors = 360176011)